docs/getting_started/command_line.rst

```diff
@@ -0,0 +1,344 @@
+
+great_expectations init
+==============================
+
+The first thing we'll want to do is add Great Expectations in our project by running the ``great_expectations init`` command in your project directory after installing Great Expectations.
+This will run you through a very short interactive experience to
+connect to your data, show you some sample expectations, and open Data Docs.
+
+.. note::
+
+  	You can install the python package by typing ``pip install great_expectations``, if you don't have it already.
+
+.. code-block:: bash
+
+    $ great_expectations init
+      ...
+
+You've got the entire Great Expectations directory structure now with all the code you need to get started.
+
+great_expectations docs
+==============================
+
+``great_expectations docs build``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The ``great_expectations docs build`` command builds your Data Docs site.
+You'll use this any time you want to view your expectations and validations in a web browser.
+
+.. code-block:: bash
+
+    $ great_expectations docs build
+    Building Data Docs...
+    The following Data Docs sites were built:
+    - local_site:
+       file:///Users/dickens/my_pipeline/great_expectations/uncommitted/data_docs/local_site/index.html
+
+great_expectations suite
+==============================
+
+All command line operations for working with expectation suites are here.
+
+``great_expectations suite list``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The ``great_expectations suite new`` command creates expectation suites.
+Running ``great_expectations suite list`` by itself gives a list of available expectation suites:
+
+.. code-block:: bash
+
+    $ great_expectations suite list
+    3 expectation suites found:
+      	customer_requests.warning
+      	customer_requests.critical
+      	churn_model_input
+
+``great_expectations suite new``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Create a new expectation suite.
+Just as writing SQL queries is far better with access to data, so are writing expectations.
+These are best written interactively against some data.
+
+To this end, this command interactively helps you choose some data, creates a sample suite, and opens up Data Docs.
+
+.. important::
+
+    The sample suites generated **are not meant to be production suites** - they are examples only.
+
+    Great Expectations will choose a couple of columns and generate expectations about them to demonstrate some examples of assertions you can make about your data.
+
+.. code-block:: bash
+
+    $ great_expectations suite new
+    Enter the path (relative or absolute) of a data file
+    : data/npi.csv
+
+    Name the new expectation suite [npi.warning]:
+
+    Great Expectations will choose a couple of columns and generate expectations about them
+    to demonstrate some examples of assertions you can make about your data.
+
+    Press Enter to continue
+    :
+
+    Generating example Expectation Suite...
```

Comment:
Seeing "example expectation suite" is a bit surprising when I am creating a suite that I named.